AMS_DAS.md

```diff
@@ -1,11 +1,12 @@
 # Overview
 These are the three schema that track Allocations.  DAS begat DAS2. DAS2 begat AMS.
 
-Tables: AMS, DAS, DAS2 
+Tables: 
 
-Location: Sole
+AMS can be found in the NEFSC_GARFO schema. The relevant tables are prefixed with AMS_.
+DAS can be found in the NEFSC_GARFO schema, the relevant tables are prefixed with DAS_.
+DAS2 have been removed from SOLE and cannot be accessed through the GARFO_NEFSC dblink.  
```

Comment:
would suggest changing to archived and potentially reference cold storage.

Reply:
[725b86b](https://github.com/NEFSC/READ-SSB-Lee-metadata/pull/27/commits/725b86be5c2f1f6e5c8e6b812d59ed563fda9ac1)